app/src/main/java/org/ole/planet/myplanet/ui/team/teamTask/AdapterTask.kt

```diff
@@ -31,11 +30,11 @@ class AdapterTask(private val context: Context, private val realm: Realm, privat
         list?.get(position)?.let {
             rowTaskBinding.checkbox.text = it.title
             rowTaskBinding.checkbox.isChecked = it.completed
-            rowTaskBinding.deadline.text = context.getString(R.string.deadline_colon) +
-                    formatDate(it.deadline) + if (it.completed) {
-                context.getString(R.string.completed_colon) + formatDate(it.completedTime)
+            if (it.completed) {
+                rowTaskBinding.deadline.text = context.getString(R.string.deadline_colon, formatDate(it.deadline))
             } else {
-                ""
+                rowTaskBinding.deadline.text =context.getString(R.string.two_strings,
+                    context.getString(R.string.deadline_colon, formatDate(it.deadline)), context.getString(R.string.completed_colon, formatDate(it.deadline)))
```

Comment:
return this back to initial condition as current update shows completed date even though the task is not completed
`initial`
![Screenshot_20240717_184300](https://github.com/user-attachments/assets/b56bd668-2215-490a-a679-061118329209)

`current branch`
![Screenshot_20240717_184205](https://github.com/user-attachments/assets/88169ba5-ae8b-42ad-ab8a-9815bb0792c9)

Reply:
Sorry, I had the if condition backwards. Should have fixed it though!